DataSetDocker/Dockerfile

```diff
@@ -61,29 +61,35 @@ RUN conda install --quiet --yes "tornado<5.0.0"
 RUN conda install --quiet --yes pip
 RUN pip install py2nb
 
+RUN chown -R $NB_USER /tmp/dataset
+
+# Creating scripts for updating and making examples
 ARG UPD_EX=/usr/local/bin/update_repo_and_make_examples.sh
 
 RUN touch $UPD_EX && \
     chmod a+x $UPD_EX &&\
     echo "#!/bin/bash" >> $UPD_EX && \
-    echo "cd /tmp/dataset && git checkout master && git pull" >> $UPD_EX && \
+    echo "cd /tmp/dataset && git reset --hard && git clean -fd && git checkout . && git pull" >> $UPD_EX && \
     echo "find /tmp/dataset/python/examples -name '*.py' -exec py2nb {} \; ;" >> $UPD_EX && \
     echo "if [ ! -d /home/$NB_USER/work/examples ]; then mkdir /home/$NB_USER/work/examples; fi;" >> $UPD_EX && \
     echo "if [ -d /tmp/dataset/python/examples ] && [ -d /home/$NB_USER/work/examples ]; then find /tmp/dataset/python/examples -name '*.ipynb' -exec mv {} /home/$NB_USER/work/examples \; ; fi;" >> $UPD_EX && \
     echo "chmod -R a+rwx /home/$NB_USER/work" >> $UPD_EX && \
-    echo "chmod -R a+rwx /tmp/dataset/python/examples" >> $UPD_EX && \
-    echo "chmod -R a+rwx /tmp/dataset/.git" >> $UPD_EX && \
     echo "chown -R $NB_USER /home/$NB_USER/work" >> $UPD_EX && \
-    echo "chown -R $NB_USER /tmp/dataset/python/examples" >> $UPD_EX && \
-    echo "chown -R $NB_USER /tmp/dataset/.git" >> $UPD_EX && \
     echo "exit 0" >> $UPD_EX
 
-RUN bash -x $UPD_EX
-
+#avoid using passwords for jupyter lab
 RUN sed -i "s/jupyter lab/jupyter lab --NotebookApp.token='' --NotebookApp.password=''/" /usr/local/bin/start-notebook.sh
 
+# Copy datafiles NOTE: files should be in the docker build directory
+COPY PG3_4844_event.nxs PG3_4866_event.nxs /home/$NB_USER/work/examples/
```

Comment:
Did you consider running wget directly as a docker command to obtain the files?

Reply:
Yes, it is my basic idea. I just need a URL.